enhancements/update/clusterversion-history-pruning.md

```diff
@@ -0,0 +1,166 @@
+---
+title: clusterversion-history-pruning
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - N/A
+creation-date: 2022-06-10
+last-updated: 2022-06-10
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-661
+---
+
+# ClusterVersion History Pruning
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] Operational readiness criteria is defined
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+CVO currently maintains a 50 entry queue of installed versions.
+Once the queue is full CVO simply removes the first entry which can result in the removal of useful cluster version information such as the initial installed version.
```

Comment:
nit: "first" -> "oldest" to be more specific about time being the ranking metric (vs. array index or whatever)?

Reply:
Expanded to more accurately reflect current processing.